clients/ctl/admin-ui/src/features/system/DescribeSystemStep.tsx

```diff
@@ -204,7 +173,7 @@ const DescribeSystemStep = ({ onCancel, onSuccess, abridged }: Props) => {
                 type="submit"
                 variant="primary"
                 size="sm"
-                disabled={!dirty}
+                disabled={isEditing ? false : !dirty}
```

Comment:
Nit: could use a comment explanation. At first I thought this was wrong b/c we shouldn't submit a no-op update, but then I realized you need this to move to the declare step.